doc-Managing_Authentication/topics/LDAP.adoc

```diff
@@ -24,10 +23,10 @@ To configure {product-title_short} to use LDAP for authentication:
 . Select the *Authentication* tab.
 . Select a *Session Timeout* to set the period of inactivity before a user is logged out of the console.
 . Select *LDAP* or *LDAPS* from the *Mode* list. This exposes additional required parameters under *LDAP Settings*.
-. Configure your *LDAP Settings* (the following example configures an IdM LDAP server):
-* Use *LDAP Host Names* to specify the fully qualified domain names of your LDAP servers. {product-title_short} will search each host name in order until it finds one that authenticates the user. Note, {product-title_short} supports using a maximum of three possible *LDAP Host Names*.
-* Use *LDAP Port* to specify the port for your LDAP server. The default is 389 for LDAP and 636 for LDAPS.
-* From the *User Type* list, select one of the following and configure the values for your LDAP server:
+. Configure your *LDAP Settings* (the following example configures an IdM directory server):
+* Use *LDAP Host Names* to specify the fully qualified domain names of your directory servers. {product-title_short} will search each host name in order until it finds one that authenticates the user. Note, {product-title_short} supports using a maximum of three possible *LDAP Host Names*.
```

Comment:
Not sure if it's worth making this note into a note box? Just a thought.

Reply:
I think the note box would be enormous and would break PDFs, so I'm going to leave it as is. But not a bad idea if there's another solution...

Reply:
OK, makes sense.